### PR TITLE
update README.md for better understanding when integrating

### PR DESCRIPTION
**Pull request by doemEINS**
I had issues to understand the README.md when I was integrating the valetudo-map-card into my Home Assistant setup. So I adapted some expressions and hope this helps others to understand right from the beginning. 

Feel free to edit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # Lovelace Valetudo Map Card
 
-Draws the map available from a Xiaomi Vacuum cleaner flashed with [Valetudo](https://github.com/Hypfer/Valetudo) in a [Home Assistant](https://www.home-assistant.io/) Lovelace card.
+Draws the map that is provided by [ICantBelieveItsNotValetudo](https://github.com/Hypfer/ICantBelieveItsNotValetudo) from a Xiaomi vacuum cleaner, that is rooted and flashed with [Valetudo](https://github.com/Hypfer/Valetudo) in a [Home Assistant](https://www.home-assistant.io/) Lovelace card.
```
Comment (TheLastProject) [inline README.md]: This component doesn't use ICantBelieveItsNotValetudo though, it draws the map itself

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 ```yaml
 resources:
   - type: js
     url: /community_plugin/lovelace-valetudo-map-card/valetudo-map-card.js
 ```
 
-configuration.yaml:
-Valetudo officially supports MQTT, with the preferred example configuration as follows. Unfortunately Home Assistant does not support authentication via MQTT. See below for an alternative example configuration using Valetudo's REST API (unsupported) if you prefer not to use MQTT or require authentication.
+`configuration.yaml`:
+[Valetudo officially supports MQTT](https://github.com/Hypfer/Valetudo/wiki/Home-Assistant-Integration), with the preferred example configuration as follows. Unfortunately Home Assistant does not support authentication via MQTT. See below for a deprecated alternative example configuration using Valetudo's REST API (unsupported) if you prefer not to use MQTT or require authentication.
```
Comment (TheLastProject) [inline README.md]: The first paragraph is indeed useful. Sadly, it also tells you to use ICantBelieveItsNotValetudo, which is incorrect for this addon. So we can't really link to this page as-is.

File: README.md [hunk 3]
```diff
@@ -91,3 +91,5 @@ recorder:
 | rotate | number | 0 | Value to rotate the map by (default is in deg, but a value like `2rad` is valid too)
 | crop | Object | {top: 0, bottom: 0, left: 0, right: 0} | Crop the map
 | min_height | string | 0 | The minimum height of the card the map is displayed in, regardless of the map's size itself. Suffix with 'w' if you want it to be times the width (ex: 0.5625w is equivalent to a picture card's 16x9 aspect_ratio)
+
+Colors can be any hex code in the card config, e.g. `path_color = '#FF0000'`.
```
Comment (TheLastProject) [inline README.md]: They can actually be any valid CSS value, so also `red` or `rgb(255,255,255)` or `rgba(255,255,255,0.8)`.

File: README.md (hunk 1 above)
Comment (doemEINS) [reply README.md]: Okay, I thought valetudo-map-card would use the map provided by ICantBelieveItsNotValetudo, but as it does not, I removed that phrase.

File: README.md (hunk 2 above)
Comment (doemEINS) [reply README.md]: Fair enough, link removed.

File: README.md (hunk 3 above)
Comment (doemEINS) [reply README.md]: Extended that as well.

**Comment by TheLastProject** [comment]
Definitely a good improvement, thanks!

**Pull request merged**